### PR TITLE
CHANGELOG for 0.0.3

### PR DESCRIPTION
**Pull request by angelozerr**
CHANGELOG for 0.0.3

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by jeffmaury** [comment]
Not sure to understand as with this change there won't be an 0.0.3 published ever !!!!

**Comment by angelozerr** [comment]
> Not sure to understand as with this change there won't be an 0.0.3 published ever !!!!

@fbricon published yesterday a new version of vscode-quarkus but version was not incremented in pom.xml that's why I update the pom.xml directly in 0.0.4.

**Comment by jeffmaury** [comment]
Not sure to understand why the changelog mentions 0.0.3 then but that may be related to VSCode extension

**Comment by angelozerr** [comment]
@fbricon can we merge it?

File: quarkus.ls/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Quarkus Language Server Changelog
 
+## 0.0.3-SNAPSHOT (October 23, 2019)
+
+### Enhancements
+
+ * Provide default method when extending QuarkusLanguageClientAPI. See [#120](https://github.com/redhat-developer/quarkus-ls/issues/120)
+ * Support glob pattern to exclude unknown validation. See [#79](https://github.com/redhat-developer/quarkus-ls/issues/79)
```
Comment (fbricon) [inline quarkus.ls/CHANGELOG.md]: exclude unknown properties from validation

Comment (angelozerr) [reply quarkus.ls/CHANGELOG.md]: @fbricon fixed.

**Comment by angelozerr** [comment]
@fbricon can we merge it now?

**Pull request merged**